Sofa/Component/Mapping/Linear/src/sofa/component/mapping/linear/BarycentricMappers/BarycentricMapperEdgeSetTopology.inl

```diff
@@ -55,26 +55,19 @@ typename BarycentricMapperEdgeSetTopology<In, Out>::Index BarycentricMapperEdgeS
 }
 
 template <class In, class Out>
-type::vector<Edge> BarycentricMapperEdgeSetTopology<In,Out>::getElements()
+auto BarycentricMapperEdgeSetTopology<In,Out>::getElements() -> type::vector<Edge>
 {
     return this->m_fromTopology->getEdges();
 }
 
 template <class In, class Out>
-type::vector<SReal> BarycentricMapperEdgeSetTopology<In,Out>::getBaryCoef(const Real* f)
+auto BarycentricMapperEdgeSetTopology<In,Out>::getBarycentricCoefficients(const std::array<Real, MappingData::NumberOfCoordinates>& barycentricCoordinates) -> std::array<Real, Edge::NumberOfNodes>
```

Comment:
I wonder if juste writing 2 instead of Edge::NumberOfNodes wouldn't make the code more readable. 
Unless it is common for an Edge  to have more than two nodes... an Edge in an hyperspace ?